File: java/oop/AccountAppWithException/src/com/techlabs/account/test/TestAccount.java
package com.techlabs.account.test;

import com.techlabs.account.Account;

public class TestAccount {
	public static void main(String[] args){
		Account account1=new Account(101, "Chandan",1000.0);
		
		account1.deposit(500);
		printDetails(account1);
		try {
			System.out.println("----------------------\nTrying To withdraw");
			//account1.withdraw(1000);
			account1.withdraw(1200);
		} catch (Exception e) {
			System.out.println(e.getMessage()+"\n-------------------------------------");
		}
		printDetails(account1);
	}
	
	private static void printDetails(Account account){
		System.out.println("Account No: "+account.getAccountNo()
				+"\nName: "+account.getName()
				+"\nBalance: "+account.getBalance());
	}
}

File: dotNet/CSharp/AccountAppWithCustomException/AccountAppWithCustomException/Test/TestAccount.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AccountAppWithCustomException.Test
{
    class TestAccount
    {
        static void Main(string[] args)
        {
            Account account1 = new Account(101, "chandan", 1000);

            try
            {
                account1.Withdraw(600);
            }
            catch (Exception e)
            {
                Console.WriteLine(e.Message);
            }
        }
    }
}
